docs/html/user_guide.rst

```diff
@@ -242,6 +242,10 @@ organisation and use that everywhere. If the thing being installed requires
 "helloworld" to be installed, your fixed version specified in your constraints
 file will be used.
 
+Constraints file can be served via a URL, e.g.
+http://example.com/constraints.txt instead of only a local file, so that your
+organization can provide constraints files online from a centraliazed place.
```

Comment:
This is the same for the requirments file format, and from the flow of text I feel this block should be added to the section above (on requirements files) and referenced here instead.

Reply:
Ah, thanks! I did not know requirements file can be served online.